Userland/Libraries/LibJS/Tests/builtins/Number/Number.js

```diff
@@ -17,11 +17,21 @@ test("constructor without new", () => {
     expect(Number("Infinity")).toBe(Infinity);
     expect(Number("+Infinity")).toBe(Infinity);
     expect(Number("-Infinity")).toBe(-Infinity);
+    expect(Number("Infi" + "nity")).toBe(Infinity);
+    expect(Number("-Infi" + "nity")).toBe(-Infinity);
```

Comment:
I don't see much value in adding these, let's just focus on which values are actually tested instead of assembling strings, which is already extensively tested elsewhere :)

Reply:
I got test262 failures which tried this but that turned out to be something else, they're gone